doc/working-with-the-micro-integrator.md

```diff
@@ -91,7 +92,7 @@ required configurations to modify the defaults in the Micro Integrator base imag
 ### Trying out a sample scenario
 
 For instructions on trying out a simple use case using the micro Integrator profile, see
-[Hello world sample](examples/hello-world).
+[Hello world sample](../examples/hello-world).
```

Comment:
'Trying out a sample scenario' in 'Micro Integrator with Kubernates' has more information. Shall we add that content here as well?

Reply:
Added in https://github.com/wso2/micro-integrator/pull/102/commits/ca13a708cd439ba4abbaea4338a25232d5f015f3